### PR TITLE
fix(FileViewer): #MAG-203 : fix css and refresh file

### PR DESCRIPTION
**Pull request by Rseuret**
## Describe your changes
- Fixing css for csv and other big file
- Fixing refresh fileViewer directive when same file type is used

## Checklist tests
-Check if you can edit lool file and other file from workspace

## Issue ticket number and link

https://jira.support-ent.fr/browse/MAG-203

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: src/main/resources/public/sass/global/components/directives/_file-viewer.scss [hunk 1]
```diff
@@ -1,13 +1,19 @@
 .file-viewer{
   max-height: 50vh;
-  min-width: 60vw;
-  overflow-y: auto;
-
+  height: 100%;
+  //min-width: 30vw;
```
Comment (flmartineau) [inline src/main/resources/public/sass/global/components/directives/_file-viewer.scss:4]: Enlève la ligne commentée

File: src/main/resources/public/template/board-read.html [hunk 2]
```diff
@@ -30,14 +30,14 @@
         <!-- Card display -->
         <card-display
                 card="vm.card"
-                board="vm.board">
+                board="vm.board"
+                change-page-eventer="vm.changePageSubject">
```
Comment (flmartineau) [inline src/main/resources/public/template/board-read.html:34]: Tu peux retirer l'eventer

File: src/main/resources/public/ts/directives/card-display/card-display.directive.ts [hunk 3]
```diff
@@ -5,6 +5,7 @@ import {Card, Board} from "../../models";
 import {DateUtils} from "../../utils/date.utils";
 import {RESOURCE_TYPE} from "../../core/enums/resource-type.enum";
 import {I18nUtils} from "../../utils/i18n.utils";
+import {Subject} from "rxjs";
```
Comment (flmartineau) [inline src/main/resources/public/ts/directives/card-display/card-display.directive.ts:8]: Retire l'import

File: src/main/resources/public/ts/directives/card-display/card-display.directive.ts [hunk 4]
```diff
@@ -84,7 +84,8 @@ function directive() {
         templateUrl: `${RootsConst.directive}card-display/card-display.html`,
         scope: {
             card: '=',
-            board: '='
+            board: '=',
```
Comment (flmartineau) [inline src/main/resources/public/ts/directives/card-display/card-display.directive.ts:87]: Virgule en trop

File: src/main/resources/public/ts/directives/card-list/card-preview-image.directive.ts [hunk 5]
```diff
@@ -4,6 +4,7 @@ import {RootsConst} from "../../core/constants/roots.const";
 import {Card, Board} from "../../models";
 import {RESOURCE_TYPE} from "../../core/enums/resource-type.enum";
 import {EXTENSION_FORMAT} from "../../core/constants/extension-format.const";
+import {Subject} from "rxjs";
```
Comment (flmartineau) [inline src/main/resources/public/ts/directives/card-list/card-preview-image.directive.ts:7]: Import a enlever + , dans le scope

File: src/main/resources/public/ts/directives/card-list/card-preview-image.html [hunk 6]
```diff
@@ -7,7 +7,9 @@
                  file="vm.card.resource"
                  content-type="vm.card.resource.previewRole()"
                  has-download="vm.board.myRights.contrib !== undefined"
-                 has-edit="vm.board.myRights.publish !== undefined"></file-viewer>
+                 has-edit="vm.board.myRights.publish !== undefined"
+                 change-page-eventer="vm.changePageEventer"
```
Comment (flmartineau) [inline src/main/resources/public/ts/directives/card-list/card-preview-image.html:11]: Eventer a enlever

File: src/main/resources/public/ts/directives/file-viewer/file-viewer.directive.ts [hunk 7]
```diff
@@ -8,6 +8,8 @@ import {safeApply} from "../../utils/safe-apply.utils";
 import {FileViewModel} from "./FileViewerModel";
 import {hasRight} from "../../utils/rights.utils";
 import {Board} from "../../models";
+import {Subject} from "rxjs";
```
Comment (flmartineau) [inline src/main/resources/public/ts/directives/file-viewer/file-viewer.directive.ts:11]: import

File: src/main/resources/public/ts/directives/file-viewer/file-viewer.directive.ts [hunk 8]
```diff
@@ -173,7 +181,8 @@ function directive(){
             file: '=',
             contentType: '=',
             hasDownload: '=',
-            hasEdit: '='
+            hasEdit: '=',
```
Comment (flmartineau) [inline src/main/resources/public/ts/directives/file-viewer/file-viewer.directive.ts:184]: , en trop

File: src/main/resources/public/ts/directives/file-viewer/file-viewer.html [hunk 9]
```diff
@@ -21,8 +27,8 @@ <h2 class="no-margin">[[vm.file.name]]</h2>
         <audio ng-if="vm.contentType === 'audio'" controls class="render" ng-src="[[vm.file.link]]"></audio>
         <video ng-if="vm.contentType==='video' && vm.isStreamable()" controls class="render" ng-src="[[vm.file.link]]" data-document-id="[[vm.file._id]]" data-document-is-captation="[[vm.file.metadata.captation]]"></video>
         <video ng-if="vm.contentType==='video' && !vm.isStreamable()" controls class="render" ng-src="[[vm.file.link]]" data-document-id="[[vm.file._id]]"></video>
-        <pdf-viewer ng-if="vm.contentType === 'pdf'" ng-src="[[vm.file.link]]"></pdf-viewer>
-        <pdf-viewer ng-if="vm.isOfficePdf()" ng-src="[[vm.previewUrl()]]" scroller="'#embeded-viewer'"></pdf-viewer>
+        <pdf-viewer ng-if="vm.contentType === 'pdf'" ng-src="[[vm.file.link]]" reload-pdf="vm.changePageEventer"></pdf-viewer>
```
Comment (flmartineau) [inline src/main/resources/public/ts/directives/file-viewer/file-viewer.html:30]: retire reload-pdf

File: src/main/resources/view-src/magneto.html [hunk 10]
```diff
@@ -18,6 +18,7 @@
 	</head>
 
 	<body ng-controller="MainController as vm" class="magneto">
+	<div class="navbar"></div>
```
Comment (flmartineau) [inline src/main/resources/view-src/magneto.html:21]: C'est pour quoi ?

**Review by FlorentMr — APPROVED** [review]

**Review by flmartineau — APPROVED** [review]

**Pull request merged**